src/services/scsc.py

```
@@ -214,39 +214,18 @@ async def update_global_status(
-                    },
-                )
-                sig_res = await mq_client.send_discord_bot_request(
-                    action_code=3005,
+                    action_code=3002,
                     body={
                         "category_name": f"{scsc_global_status.year}-{map_semester_name.get(scsc_global_status.semester)} SIG Archive"
                     },
                 )
-                pig_res = await mq_client.send_discord_bot_request(
-                    action_code=3005,
+            if not pig_res:
```

Ruff (0.15.9)

[error] Undefined name `pig_res`

(F821)